lapack/gonum/dlange.go

```diff
@@ -73,14 +75,14 @@ func (impl Implementation) Dlange(norm lapack.MatrixNorm, m, n int, a []float64,
 			value = math.Max(value, sum)
 		}
 		return value
+	default:
```

Comment:
There is a check at the beginning for valid `norm` input, so `norm` must be one of the four possible values. That's why I removed the default panic and replaced it with Frobenius. I'm inclined to change Dlansb to do the same.

Also, it seems to me that Fortran returns an uninitialized `VALUE` rather than 0 if `NORM` is not valid.

Reply:
Sorry, you are right. And, yes, the Fortran does return the uninitialised value (I forget about other languages' sloppiness on this).